### PR TITLE
Update python_ci.yml

### PR DESCRIPTION
**Pull request by kmilo9999**

### CODE REVIEW

File: .github/workflows/python_ci.yml [hunk 1]
```diff
@@ -4,12 +4,12 @@ jobs:
   python_run_scripts:
     strategy:
       matrix:
-        version: [3.9, 3.10, 3.11]
-    runs-on: ubuntu-latest
+        python: [3.9.12, 3.10.8, 3.11.10]
```
Comment (hollandjg) [inline .github/workflows/python_ci.yml:7]: Are the bug fix version numbers required here, or is:
```suggestion
        python: [3.9, 3.10, 3.11]
```
sufficient?

File: .github/workflows/python_ci.yml [hunk 2]
```diff
@@ -4,12 +4,12 @@ jobs:
...
+    runs-on: ubuntu-20.04
```
Comment (hollandjg) [inline .github/workflows/python_ci.yml:8]: Is this change necessary? Was there an error running on ubuntu-latest? (And if so, does it work on ubuntu-22.04, the newest LTS version?)

**Review by hollandjg — COMMENTED** [review]
Thanks, Camilo! I have a couple of questions – what was necessary here?

**Comment by kmilo9999** [comment]
@hollandjg  @cpaniaguam  Im gonna close this PR. It's pointing to a wrong branch.
